DigitalLearningSolutions.Web/ViewModels/LearningMenu/ContentViewerViewModel.cs

```diff
@@ -1,50 +1,67 @@
 ﻿namespace DigitalLearningSolutions.Web.ViewModels.LearningMenu
 {
+    using System.Text.RegularExpressions;
+    using DigitalLearningSolutions.Data.Models.TutorialContent;
     using DigitalLearningSolutions.Web.Helpers;
     using Microsoft.Extensions.Configuration;
 
     public class ContentViewerViewModel
     {
-        public string HtmlSource { get; }
-        public string ScormSource { get; }
+        public TutorialContent TutorialContent { get; }

```

Comment:
Do you need to store the entire tutorial content? 🙂

Reply:
I use all of the `TutorialContent` to make the source URL, but then don't use too much of it. I suppose I could take it in the constructor to make this URL, but then discard it afterwards if you like? 🙂

Reply:
Yes I think I'd prefer that. I think it's best if there's nothing publically accessible which isn't necessary. I think that makes the code a bit easier to understand 🙂

Reply:
Done in 141cbd5 🙂